hydra-cluster/src/CardanoClient.hs

```diff
@@ -40,7 +40,7 @@ buildScriptAddress script networkId =
 buildRaw :: [TxIn] -> [TxOut CtxTx] -> Lovelace -> Either TxBodyError TxBody
 buildRaw ins outs fee =
   createAndValidateTransactionBody $
-    defaultTxBodyContent
+    defaultTxBodyContent BabbageEra
```

Comment:
I would suggest to also shim `defaultTxBodyContent` like we did it for `createAndValidateTransactionBody`